src/follow-me/render.php

```diff
@@ -92,25 +97,83 @@
 	)
 );
 
+$header_image = $actor->get_image();
+$has_header   = ! empty( $header_image['url'] ) && str_contains( $attributes['className'] ?? '', 'is-style-profile' );
+
+$stats = array(
+	'posts'     => count_user_posts( $user_id, 'post', true ),
+	'followers' => Followers::count_followers( $user_id ),
+	'following' => null,
```

Comment:
```suggestion
	'following' => 0,
```

why not always show it?

Reply:
I figured we leave it off while users can't follow others. It would always be 0.

Reply:
why not removing it completely then?

Reply:
Fixed in [767e94a](https://github.com/Automattic/wordpress-activitypub/pull/1753/commits/767e94ad28ae9e99e2e635d22498e8c2c564caf2)